### PR TITLE
Add a simple script show the API of the Gtk::ListStore

### PR DESCRIPTION
**Pull request by cedlemo**
I don't know if you could find this usefull but I have written this very simple script in order to see how the Gtk::ListStore methods works so I share.

### CODE REVIEW

File: gtk3/sample/misc/list_store_usage.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    print iter[1] + "  "
+  end
+  print "\n"
+  puts ""
+end
+
+def load_list
+  model = Gtk::ListStore.new(TrueClass, String)
+
+  (1..4).each do |i|
```
Comment (kou) [inline gtk3/sample/misc/list_store_usage.rb]: `4.times` is better. You don't need to do `#{i - 1}` with it.

**Comment by kou** [comment]
I think that it's useful. :-)
I want to merge it after you apply my comment.

**Comment by cedlemo** [comment]
Done!

**Comment by kou** [comment]
Merged!

**Pull request merged**